js/skip-nav.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/* global require, module, document */
+
+var $ = require('jquery');
+
+function Skipnav(selector) {
+  var self = this;
+  self.$link = $(selector);
+  self.$target = self.findTarget();
+  $(document.body).on('click, keyup', self, this.focusOnTarget.bind(this));
+};
+
+Skipnav.prototype.findTarget = function() {
+  return $('main').find('a, button, :input, [tabindex]').filter(':visible')[0];
```

Comment:
Prefer to minimize hard-coded selectors in generic UI components. How about moving "main", or even all of `$target`, to a parameter of the constructor?